client/layout/masterbar/logged-in.jsx

```diff
@@ -107,6 +117,7 @@ class MasterbarLoggedIn extends React.Component {
 		} else if ( 'reader' !== this.props.section || 'sidebar' === this.props.currentLayoutFocus ) {
 			// when reader is not focused or sidebar is open, focus to reader's content. Else, open reader's sidebar.
 			this.props.setNextLayoutFocus( 'content' );
+			this.props.setNextLayoutFocus( 'sidebar' );
```

Comment:
Can we remove the layout focus on `content` if we set it to `sidebar` anyway?

Reply:
Could point, removed.